### DISCUSSION TITLE
Any way to have main carousel watchOverflow watch the thumbnails carousels state.

### DISCUSSION THREAD

**Discussion by inquiztr** [nolimits4web/swiper#5752] [Ideas]
I am using the main carousels arrows positioned to the left and right of the thumbnails carousel due to the bug in thumbnails carousels arrow functionality. But by doing this the arrows do not know when to hide/disable in response to the thumbnails carousel.
Is there any suggestion on how to use the main gallery arrows but still have them hide when the thumbnails gallery dont need arrows (since all thumbnails properly fit on the screen)

**Comment by Arhell** [comment]
If I understand your question correctly, then you can use breakpoints
https://swiperjs.com/swiper-api#param-breakpoints
or using css to do what you need by rewriting the functionality
